guides/micronaut-rabbitmq-rpc/bookrecommendation/groovy/src/main/groovy/example/micronaut/BookRecommendation.groovy

```diff
@@ -0,0 +1,19 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import groovy.transform.EqualsAndHashCode
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@CompileStatic
+@EqualsAndHashCode
+@Introspected
+class BookRecommendation {
+
+    final String name;
+
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.